### ISSUE TITLE
Dark theme colors in hex

### ISSUE THREAD

**Issue by gidsi** [hrkfdn/ncspot#114]
Hey,

I really like the project thanks for the nice work :)

I've been working on creating themes for `ncspot` and i think i found a strange bug.

While `#080808` (or e.g. `black`) provides you with the correct color, using `#080807` leads to a pink'ish tone on my system (i haven't checked other systems yet):
![2020-01-10-16:18:07](https://user-images.githubusercontent.com/7644803/72163850-de099d00-33c4-11ea-9540-f82df56f4806.png)

That seems to be the problem for every color if one of RGB is below `8` and no other value is above  or equal `2B`. So e.g. `#002B00` is working, while `#002A00` is pink.

Here is the theme used in the above picture:
```
[theme]
background = "#080808"
primary = "#080807"
secondary = "#080807"
title = "#080807"
playing = "#080807"
playing_bg = "#080807"
highlight = "#080807"
highlight_bg = "#080807"
error = "#080807"
error_bg = "#080807"
statusbar = "#080807"
statusbar_progress = "#080807"
statusbar_bg = "#080807"
cmdline = "#080807"
cmdline_bg = "#080807"
```

**Comment by hrkfdn** [comment]
Mhh, I can't really reproduce this on my system using `gnome-terminal`. So there are many factors at play here, such as the terminal emulator or what `$TERM` is set to. Could you share some more details?

**Comment by gidsi** [comment]
> Mhh, I can't really reproduce this on my system using `gnome-terminal`. So there are many factors at play here, such as the terminal emulator or what `$TERM` is set to. Could you share some more details?

I can't confirm it on different systems. 
I tried with identical version/os/$TERM/terminal emulator, the problem doesn't show up again. 

I would assume a glitch/problem with the system and leave it be, i'll tell you if i've found it :)

**Comment by gidsi** [comment]
Okay, i can confirm on both other systems too.

What happens is that i can use the theme on first run, after the first run i can use is as many times as i want to in the same terminal and the bug won't occur. 

The moment i open it in another terminal (regardless if i close the original one) it changes the colors as mentioned above (but to different colors too, pink / yellow / orange so far).

Changing the theme doesn't help, restarting everything doesn't help either, the only way i can correct this is by deleting the config.toml file and start ncspot to let it be recreated, recreating it by myself doesn't work.

**Comment by gidsi** [comment]
> Mhh, I can't really reproduce this on my system using `gnome-terminal`. So there are many factors at play here, such as the terminal emulator or what `$TERM` is set to. Could you share some more details?

What i've tested so far:
$TERM=xterm-256color
$TERM=rxvt-unicode-256color

Terminal emulators:
alacritty
urxvt

Packages:
https://aur.archlinux.org/packages/ncspot-git/
https://aur.archlinux.org/packages/ncspot/

System:
Arch Linux 5.4.8-arch1-1

**Comment by elianiva** [comment]
![image](https://user-images.githubusercontent.com/51877647/85653676-8f805e80-b6d7-11ea-8ad0-21ecd852135a.png)

Same issue here, I'm using Alacritty with `$TERM` set to alacritty, setting it to xterm-256color and rxvt-unicode-256color gives me the same result. The background used is `#292d3e` which should look very dark blue-ish background.
The very blue color is `#212333` which should look close to black but lighter but instead it shows blue.

Details: 
Artix Linux 5.7.4-artix1-1
Alacritty 0.4.3
Ncsport 0.1.3 (https://aur.archlinux.org/packages/ncspot/)

**Comment by hhsnopek** [comment]
I similarly have this issue with the color: `#24364b`. The background of `ncspot` (color above) should match the bottom most background color.

<img width="590" alt="Screen Shot 2020-06-25 at 1 47 44 PM" src="https://user-images.githubusercontent.com/2363940/85780669-7c2abe80-b6ea-11ea-9987-d8b27dcce5d9.png">

Details:
```
$TERM=xterm-256color
alacritty version 0.4.2 (f68de37)
ncspot version HEAD (0f08a56)
```

**Comment by benpye** [comment]
I found that building with the termion backend for pancurses appears to fix custom colors with Alacritty. YMMV.

**Comment by hrkfdn** [comment]
Fixed by #894, if not feel free to re-open
